task-header.css

```
@@ -89,11 +94,15 @@ body {
 }
 	
 	.infobar {
-		margin-top: 15%;
+		margin-top: 20%;
 }
 		
 	.social {
-		background-color: #a4b6c0);
+		background-color: #a4b6c0;
+		border-left: 4px solid rgb(87, 118, 128);
+		border-bottom: 4px solid rgb(87, 118, 128);
+		border-right: 4px solid rgb(87, 118, 128);
+		border-top: 4px solid rgb(87, 118, 128);
```

Since you're applying the same border to every side you can just do this:
```suggestion
		border: 4px solid rgb(87, 118, 128);
```

---

Literally just spotted this when looking at the previous comment - no idea why I did that. All updated.